SonicWall/sonicwall-fw/tests/test_syslog_website_accessed.json

```diff
@@ -0,0 +1,112 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "SonicWall Firewall",
+        "dialect_uuid": "ee0b3023-524c-40f6-baf5-b69c7b679887"
+      }
+    },
+    "message": "CEF:0|SonicWall|NSa 4500|7.0.1-1234-R5678|97|Syslog Website Accessed|4|cat=1024 gcat=2 smac=00:11:22:33:44:55 src=12.3.123.123 spt=60000 deviceInboundInterface=X0-V123 cs3Label=WORKSTATION cs1Label=123.456.789.123 snpt=12345 dmac=33:33:33:33:33:33 dst=222.3.4.55 dpt=444 deviceOutboundInterface=X1 cs4Label=WAN cs2Label=123.45.67.890 dnpt=444 susr=\"USER\" proto=tcp/https out=1234 in=4567 cs5Label=\"Auto\" cs1=\"WORKSTATIONS -> WEB\" app=2233 request=microsoft.com/ reason=22 Category-\"Computers\" cs6=\"Policy: Default Policy\" cnt=123456789 fw_action=\"drop\" dpi=1"
+  },
+  "expected": {
+    "message": "CEF:0|SonicWall|NSa 4500|7.0.1-1234-R5678|97|Syslog Website Accessed|4|cat=1024 gcat=2 smac=00:11:22:33:44:55 src=12.3.123.123 spt=60000 deviceInboundInterface=X0-V123 cs3Label=WORKSTATION cs1Label=123.456.789.123 snpt=12345 dmac=33:33:33:33:33:33 dst=222.3.4.55 dpt=444 deviceOutboundInterface=X1 cs4Label=WAN cs2Label=123.45.67.890 dnpt=444 susr=\"USER\" proto=tcp/https out=1234 in=4567 cs5Label=\"Auto\" cs1=\"WORKSTATIONS -> WEB\" app=2233 request=microsoft.com/ reason=22 Category-\"Computers\" cs6=\"Policy: Default Policy\" cnt=123456789 fw_action=\"drop\" dpi=1",
+    "event": {
+      "code": "97",
+      "severity": 4,
+      "category": [
+        "network"
+      ],
+      "kind": "event",
+      "type": "protocol"
+    },
+    "observer": {
+      "vendor": "SonicWall",
+      "type": "firewall",
+      "version": "7.0.1-1234-R5678",
+      "egress": {
+        "interface": {
+          "name": "X1"
+        }
+      },
+      "ingress": {
+        "interface": {
+          "name": "X0-V123"
+        }
+      }
+    },
+    "sonicwall": {
+      "fw": {
+        "dpi": true,
+        "fw_action": "drop",
```

Comment:
I think we can copy this value and add it to `event.action` and change `event.type` to `['denied']` in this case